health/health_log.c

```diff
@@ -198,6 +198,24 @@ static inline ssize_t health_alarm_log_read(RRDHOST *host, FILE *fp, const char
         int max_entries = 33, entries = 0;
         char *pointers[max_entries];
 
+        char *c = buf;
+        while (*c) {
+            if (unlikely(!isascii(*c))) {
```

Comment:
This probably needs to change from `(*c)` to  `((unsigned char) *c)`

Reply:
Hi @stelfrag, thanks! Pushed 8ec9012c1ae2896c34545312ce3b280da1a2dc1f